graphviz.spec

```diff
@@ -366,11 +395,15 @@ Static development package for %{name}.
 %setup -q
 %apply_patches
 
-autoreconf -fiv
+#cat >./version.m4 <<EOF
```

Comment:
Why add a commented-out line that does evil things?

Reply:
It's gone!